### ISSUE TITLE
Sysout-Calls in AWTPanel

### ISSUE THREAD

**Issue by oOMoeOo** [jMonkeyEngine/jmonkeyengine#539]
Could you guys please remove the Sysout-Calls in the AwtPanel class or maybe replace it with a logger. Especially when resizing the awt-panel in a swing environment, the console should be quiet.

Affected lines: 102, 115, 125, 144, 152, 154

**Comment by empirephoenix** [comment]
Oh, good find, if you could just make a pull request for this, I can merge it.

**Comment by oOMoeOo** [comment]
Ok, i guess some outputs are very useless but the one in line 201 indicates an invalid graphics object. Are there any specific requirements for using loggers? If not i'm gonna use the standard java.util.logging.

**Comment by Dokthar** [comment]
Logging is done with the standard java.util.logging you can see an example in the [Node](https://github.com/jMonkeyEngine/jmonkeyengine/blob/master/jme3-core/src/main/java/com/jme3/scene/Node.java) class.

**Comment by oOMoeOo** [comment]
When do you guys think this issue could be merged into the current or next release version? The request was merged into master in august 2016 but i still see the issue in 3.1.0-stable.

**Comment by MeFisto94** [comment]
This indeed is a fault. The problem is that your pull request did not get merged into the `3.1` branch back then, so it is only in next (`3.2`).
Could you maybe open a new PR which also applies it to the 3.1 branch?

I don't know when a new stable version will be released, though, since that alone is annoying but probably not enough for releasing a 3.1.1-stable

**Comment by oOMoeOo** [comment]
Well that confuses me. I was thinking that issues get merged from master to the current/next release branch O.o

Should i merge the commit from master to 3.1 in my personal fork and then create a pull request from oOMoeOo/v3.1 to jmonkeyengine/v3.1?

**Comment by empirephoenix** [comment]
master is for 3.2, so no 3.1 release will see any changes.

**Comment by pspeed42** [comment]
re: "Well that confuses me. I was thinking that issues get merged from master to the current/next release branch O.o"  How would that work, exactly?  We would never get to a stable release of anything ever if we merged all changes to all branches.  It can't work.

When we start a release process, we create a branch.  I think we created the 3.1 branch more than 2 years ago(?).  Sometimes when a person submitted a change to the master branch (future 3.2) then we'd also apply it to the 3.1 branch if it was appropriate.  This is very hard to keep track of, though, and it's way easier if people submitting patches clue us into where they think it should go by applying it to the appropriate branch(es).  3.1 is kind of bug fixes only now and we usually let the person submitting the PR have the first crack at filtering, I guess.  It's too easy for us to miss something otherwise... as in this case.
